src/Traits/GetDependenciesTrait.php

```diff
@@ -11,10 +11,10 @@
 
 trait GetDependenciesTrait
 {
-    protected function resolveClassMethodDependencies(array $parameters, $instance, $method)
+    protected function resolveClassMethodDependencies($instance, $method)
```

Comment:
@hans-thomas let's make some refactoring here:
1. `resolveClassMethodDependencies` should be removed
2. `getDependencies` add `array` as return type hint
3. the first arg of the `getDependencies` should be a `$instance`, the second `string $method`
4. `getDependencies` should be renamed to `resolveClassMethodDependencies`
5. SwaggerService->getConcreteRequest should check if method in controller does not exists, and return `null` if yes, otherwise - it should call `resolveClassMethodDependencies`